tests/unit/plugins/inventory/test_kubevirt.py

```diff
@@ -6,12 +6,14 @@
 
 __metaclass__ = type
 
+import copy
+import json
```

Comment:
Can you import just the needed functions from those packages? (`from ... import ...`)

Reply:
Done.